choccake-react/src/components/IngredientList.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+
+export default class Ingredients extends Component {
+  render() {
+    return (
+      <ol>
+        <ul>
```

Comment:
This ul should be an IngredientsItem component. Here is where you would map through the ingredients prop that you passed down from App.js, just like you did in App.js lines 19-26